public/checkout.js

```diff
@@ -38,14 +38,18 @@ var packeteryLoadCheckout = function( $, settings ) {
 			return rateAttrValues[ carrierRateId ][ attribute ];
 		}
 
+		var shortifyShippingRateId = function( rateId ) {
```

Comment:
Prosím shorten místo shortify (je to nestandardní synonymum shorten) a doplnit changelog.